packages/esm-patient-programs-app/src/programs/program-actions-menu.scss

```diff
@@ -0,0 +1,11 @@
+.layer {
```

Comment:
Is this CSS necessary? I don't see why this should be any different from the other actions menus we have throughout the app.